lib/daru/index.rb

```diff
@@ -96,6 +96,27 @@ def [](*key)
         v
       end
     end
+    
+    def pos *args
+      # Causes Segmentation fault in array_wrapper
+      if args.first.is_a? Range
+        return args.first.to_a
```

Comment:
@zverok Do you know how reproduce this issue? I can't figure out which spec is causing the trouble.